### PR TITLE
new testclass added 

### PR DESCRIPTION
**Pull request by testingpoint09**
Hi Sharun,

please review the code and let me know.

Thanks,
Byoganad.

### CODE REVIEW

File: src/test/java/yoganand/testclass.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package yoganand;
+
+public class testclass {
+
```
Comment (sharunsher-007) [inline src/test/java/yoganand/testclass.java:4]: please add some code

**Review by sharunsher-007 — CHANGES_REQUESTED** [review]
please add some code at main method

File: src/test/java/yoganand/testclass.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package yoganand;
+
+public class testclass {
+	
+	public static void main(String[] args) {
+		System.out.println("hi..");
```
Comment (testingpoint09) [inline src/test/java/yoganand/testclass.java:6]: code updated and pelase review sharun.

Comment (sharunsher-007) [reply src/test/java/yoganand/testclass.java:6]: done

File: src/test/java/yoganand/testclass.java (hunk 1 above)
Comment (sharunsher-007) [reply src/test/java/yoganand/testclass.java:8]: done

**Review by sharunsher-007 — APPROVED** [review]
pls merge the code

**Pull request merged**